host/3.0/buster/amd64/dotnet/dotnet-isolated/start.sh

```diff
@@ -21,6 +21,11 @@ sed -i "s/SSH_PORT/$SSH_PORT/g" /etc/ssh/sshd_config
 
 service ssh start
 
+echo -e "**WARNING**: You are using an outdated version of the Azure Functions runtime:
+https://learn.microsoft.com/en-us/azure/azure-functions/migrate-version-3-version-4
```

Comment:
any reason for not reusing /azure-functions-host/start_deprecated.sh here?

Reply:
Made some changes to see if we can call the start_deprecated.sh script from the start.sh script in order to consolidate the warning message :)